packages/desktop/src/main/index.ts

```diff
@@ -49,7 +53,26 @@ const createMainWindow = () => {
         width: 1200,
     });
 
+    // Listen for renderer crashes and attempt a reload.
+    mainWindow.webContents.on("crashed", (event, killed) => {
+        console.error("Renderer process crashed. Killed:", killed);
+        if (reloadAttempts < MAX_RELOAD_ATTEMPTS) {
+            reloadAttempts++;
+            console.log(`Reloading renderer... attempt ${reloadAttempts}/${MAX_RELOAD_ATTEMPTS}`);
+            setTimeout(() => {
+                if (mainWindow) {
+                    mainWindow.reload();
+                }
+            }, 1000);
+        } else {
+            console.error("Maximum reload attempts reached. Quitting application.");
+            app.quit();
+        }
+    });
+
     mainWindow.once("ready-to-show", () => {
+        // Reset reload counter once the renderer is successfully loaded.
+        reloadAttempts = 0;
```

Comment:
This does not appear to be resetting correctly. I tried just resetting the app three times in succession (each time waiting until it loaded successfully) and the count keeps adding instead of going back to zero

Reply:
hmm ok I will look into this

Reply:
resolved in [45ea20a](https://github.com/AllenInstitute/biofile-finder/pull/447/commits/45ea20a5e2e2f88bd759b9fa4f2922928e7ee2a8) just use the 1did-finish-load` instead